src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -22,19 +45,24 @@ export const TodoFilter: React.FC = () => {
           type="text"
           className="input"
           placeholder="Search..."
+          value={query}
+          onChange={event => handleQuery(event.target.value)}
```

Comment:
```suggestion
          onChange={handleQuery}
```
You can accept event as handler argument. But it's on your decision.